src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefault.java

```diff
@@ -19,4 +19,29 @@ public void foo() {
         case 2: i--; break;
         }
     }
+
+    public void nestedSwitch1() {
+        int i = 1;
+        switch (i) {
+            case 1: i++; break;
+            case 2: i--; break;
+            default:
+                switch (i++) { // violation, switch without "default" clause
```

Comment:
there is not need for detailed comment (it is not documentation), just `// violation` is enough.
This comment is needed to make it easier to detect violations.
Please correct.

Reply:
OK